### ISSUE TITLE
Publish Wizard displays HTML characters

### ISSUE THREAD

**Issue by PaulaTheKoalaCT** [J2Kbr/GtunerIV#47]
Before: http://imgur.com/a/QnsI1
After: https://imgur.com/a/LF16b

Also it would be nice to have &lt;u&gt; (underline), &lt;b&gt; (bold), and &lt;i&gt; (italics) instead of using HTML characters for formatting.

**Comment by J2Kbr** [comment]
u, b, i are now allowed.
Raw ampersand encode is not allowed for security reasons.
Alternative: use shortdesc tag in a commented section in the code for the script description text.

**Comment by PaulaTheKoalaCT** [comment]
I didn't really explain the issue that well, I sent you a PM on the forums explaining what I meant.

**Comment by J2Kbr** [comment]
More info:

> I seen your response on Github for my error report and I was confused for a second, but I realized that I should of explained it better.
> 
> I get that the server converts the HTML entities to characters, what I meant to say was that when you go to update a script the description will have those characters instead of the HTML entities, which is an issue because re-submitting the script will result in question marks being displayed due to the character encoding...
> 
> I was thinking that maybe you could have the characters converted back to HTML entities and/or change the encoding so the characters display properly.

**Comment by J2Kbr** [comment]
Fixed by converting back any multi-byte UTF-8 character to the respective HTML-entity. (server side fix)
